java/ql/test/query-tests/security/CWE-1004/SensitiveCookieNotHttpOnly.qlref

```diff
@@ -0,0 +1,2 @@
+query: Security/CWE/CWE-1004/SensitiveCookieNotHttpOnly.ql
+postprocess: utils/test/InlineExpectationsTestQuery.ql
```

Comment:
We should pretty-print the models as well, otherwise the test output could change when we add totally unrelated models.
```suggestion
postprocess:
  - utils/test/PrettyPrintModels.ql
  - utils/test/InlineExpectationsTestQuery.ql
```